grails-app/conf/UrlMappings.groovy

```diff
@@ -7,13 +7,8 @@ import com.unifina.domain.signalpath.Canvas
 
 class UrlMappings {
 	static mappings = {
-		"/$controller/$action?/$id?"{
-			constraints {
-				// apply constraints here
-			}
-		}
 
-		"/"(controller:"error")
+		"/**"(redirect: [uri: "https://www.streamr.com/core/", permanent: true])
```

Comment:
Can this be derived from the base url to make it more DRY and work in all environments?

Reply:
No, I've tried injecting GrailsApplication and plain ${grails.serverURL} syntax, but neither seems to work.